packages/go-nogo/src/index.ts

```diff
@@ -0,0 +1,484 @@
+import { JsPsych } from "jspsych"
+import jsPsychHtmlButtonResponse from "@jspsych/plugin-html-button-response";
+import jsPsychHtmlKeyboardResponse from "@jspsych/plugin-html-keyboard-response";
+import jsPsychInstructions from "@jspsych/plugin-instructions";
+import { trial_text, instruction_pages } from "./text";
+
+
```

Comment:
Ideal here would be to use docstring comments so that IDEs automatically read the docs in.